protocol-asserts/pom.xml

```diff
@@ -102,28 +102,6 @@
         </executions>
       </plugin>
 
-      <!-- this is only needed to make working in Eclipse more pleasant:
-          The assertj plugin already adds the generated files as a source directory, but eclipse
-          is not able to update the project based on that plugin, so we explicitly declare it a second time -->
```

Comment:
Even though most of the team doesn't use Eclipse, that's not true for everyone AFAIK. 

❓ Are you sure we should remove this?

Reply:
100% sure that nobody develops Zeebe with Eclipse? No :smile: But nobody of the core team does, and since we have no idea if anyone does, nor do we take any extraordinary measures to provide compatibility in other modules, I think this is mostly an artifact from when half of the team used Eclipse.

Since we don't take any care for Eclipse in most of the rest of the project, it's unclear why we would here? Should we for others? I would propose to remove it and think about Eclipse support when we get pushback, instead of keeping it in the off chance somebody might still be using it. Plus, we have no idea if this is still an issue with newer versions of Eclipse.

Reply:
Isn't @romansmirnov on Eclipse?

Reply:
Exactly :100: @romansmirnov is in love :heart: with eclipse :laughing:

Reply:
🌚

Reply:
Ah, I wasn't aware :smile: I'm curious if this is still necessary in newer versions of Eclipse, but for now I'm happy to drop this then. I don't care that much about this bit, it just seemed unnecessary at this point.

Reply:
Perhaps @romansmirnov can verify whether this is still useful. I'm all for deleting unnecessary config

Reply:
I will ask him, in the mean time to not block this I'll simply drop that commit.